slice/internal/core/slice.go

```diff
@@ -33,8 +33,11 @@ func SliceKeyFromWorkload(wl *kueue.Workload, podSetName kueue.PodSetReference)
 func SliceWithMetadata(wl *kueue.Workload, podSetName kueue.PodSetReference) *v1alpha1.Slice {
 	return &v1alpha1.Slice{
 		ObjectMeta: metav1.ObjectMeta{
-			Name:      SliceName(wl.Name, podSetName),
-			Namespace: wl.Namespace,
+			Name: SliceName(wl.Name, podSetName),
```

Comment:
What if we have workloads with the same names and PodSet names in different namespaces? Should we include the namespace in the name?

Reply:
Yes, what if the user creates a prebuilt workload?